nixos/modules/services/home-automation/zwave-js-ui.nix

```
@@ -0,0 +1,86 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkIf
+    mkEnableOption
+    mkOption
+    types
+    ;
+  cfg = config.services.zwave-js-ui;
+in
+{
+  options.services.zwave-js-ui = {
+    enable = mkEnableOption "zwave-js-ui";
+    serialPort = mkOption {
+      type = types.path;
+      description = ''
+        Serial port for the Z-Wave controller.
+
+        Used for permissions only; must be additionally set in the application
+      '';
+      example = "/dev/ttyUSB0";
```

```suggestion
      example = "/dev/serial/by-id/usb-example";
```

Lets recommend a stable serial device path.